gnd/src/main/res/values/strings.xml

```diff
@@ -195,4 +195,11 @@
   <!-- Label on button shown to dismiss the feature properties dialog. -->
   <string name="close_feature_properties">OK</string>
   <string name="config_load_error">Error loading config from remote server</string>
+
+  <string name="add_point">Add Point</string>
+  <string name="save_polygon">Save Polygon</string>
+  <string name="select_data_type">Choose a data type</string>
```

Comment:
-> Choose a feature type

[I know this is how it was in the mocks, but I think "feature" would be clearer here]

Reply:
renamed